exercises/practice/etl/.meta/reference/etl.bal

```diff
@@ -0,0 +1,20 @@
+public function transform(map<string[]> old) returns map<int> {
+    map<int> transformed = {};
+    [string, int][] rows = [];
+
+    foreach var scoreAsStr in old.keys() {
+        foreach var letter in old.get(scoreAsStr) {
+            var score = int:fromString(scoreAsStr);
+
+            if score is int {
+                rows.push([letter.toLowerAscii(), score]);
+            }
+        }
+    }
```

Comment:
I tried to use a query expression for this, but didn't get it working so I just did this 🤷

Reply:
I'll take that as a challenge :)